recipe/meta.yaml

```diff
@@ -30,15 +34,27 @@ test:
     - hatch_jupyter_builder
   commands:
     - pip check
-    - pip install -e ".[test]"
-    - python -m pytest -vv
+# Note windows tests supressed because of problems with utils.run(shlex.split(cmd))
+#    - python -m pytest -k "not (test_run or test_normalize_cmd)" -vv  # [win]
+    - pip install -e ".[test]" # [not (win or s390x)]
+    - python -m pytest -vv     # [not (win or s390x)]
   requires:
+    - hatchling
```

Comment:
SInce hatchling is defined as a runtime dependency, you don't need hatching here. But you might need `hatch` (https://github.com/jupyterlab/hatch-jupyter-builder/blob/74d37d9854668fea5c51b57622aa12fc72f0a61c/pyproject.toml#L35) (though I'm not sure where it's used)